.github/dependabot.yml

```diff
@@ -11,119 +11,68 @@ updates:
       interval: "weekly"
       time: "06:00"
     open-pull-requests-limit: 5 
+    groups:
+      github-action:
+        update-types:
+        - "minor"
+        - "patch"
   - package-ecosystem: gomod
     directory: /
     schedule:
       interval: "weekly"
       time: "06:02"
     open-pull-requests-limit: 5 
+    groups:
```

Comment:
Why is it necessary to add this directive?

Reply:
So we can have one patch PR raised on Monday for go dependencies update. What do you think the current frequency as secondary oncall?

Reply:
Could you do make this change in a different PR, since it's independent of the PR's topic of removing Dockerfile group updates?